### ISSUE TITLE
Ghost *and* Medium on SEO | Open Discussion

### ISSUE THREAD

**Issue by sethbergman** [OperationCode/operationcode_infra#20]
## tl;dr: In Summary

The short version? Ghost is better for serious publishing. Medium is better for when you just want to write and don't care about the rest (SEO).

Ghost is best if…

- You care about exclusive ownership and control over your content
- Building a blog to benefit your own business is important to you
- You want to use design which you choose, not someone else

Medium may be a better choice when…

- Extra advanced functionality beyond basic publishing is not needed
- You don't want to worry about promoting your blog at all

### You can also publish to your Ghost blog and then syndicate it to Medium as a distribution mechanism. Ghost is currently working on making this even easier!

Ghost is an independent non-profit organisation.

> We build publishing tools for modern journalism & blogging because we believe freedom of speech is important. Our software is released under a free open source license, our business model is completely transparent, and our legal structure means that 100% of the money we make is reinvested into making Ghost better.

> We’re building a socially responsible, sustainable business which empowers people to share important stories and ideas. We’re in this for the long haul.

_Source:_
https://ghost.org/vs/medium/

-------

## My Opinion and Proposed Plan:

1. This is not an either or question. We should maintain our own blog on Ghost to keep the benefits of SEO and control (e.g. email capture), but also syndicate / repost to Medium to take advantage of their inbuilt distribution.

2. I am working on the best method for the automation of content syndication through Medium's and Ghost's API's. Buffer may be an option as long as it doesn't negatively affect SEO. Content is King! Remember to __own your content__ by using the TLD (top level domain) as the primary author according to Google.

#### Note:
Ghost (and it's database) can be setup through Docker Compose, or we can just use Digital Ocean's prebuilt image and point the A record in our DNS. 

### Research and Recommended Reading:
- https://themeisle.com/blog/wordpress-vs-ghost-vs-medium/
- https://growthhackers.com/questions/ask-gh-would-you-move-your-company-blog-to-medium#comment-42912
- https://blog.ghost.org/content-syndication/
- http://www.canirank.com/blog/should-i-use-medium-as-my-blog/

**Comment by sethbergman** [comment]
__Update:__

Implementation recommendation based on current infrastructure movements:
https://github.com/OperationCode/operationcode_infra/issues/27

**Comment by nellshamrell** [comment]
Thank you so much for the input!  After discussing with the other maintainers, we are not doing this at this time.
